templates/zerver/help/add-custom-profile-fields.md

```diff
@@ -40,6 +42,36 @@ There are several different types of fields available.
 * **Person picker**: For selecting other users, like "Manager" or
     "Direct reports".
 
+## Display custom fields in user profile summaries
+
+Organizations may find it useful to display additional fields in a user's
+profile summary, such as pronouns, GitHub username, job title, team, etc.
+
+All field types other than "Long text" or "Person" have a checkbox option
+that controls whether to display a custom field in a user's profile summary.
+
```

Comment:
Maybe this would look better if we merged the second and third paragraph into one?